### PR TITLE
Align threshold warnings with the default CRAP limit

### PR DESCRIPTION
**Pull request by fabian-barney**
## Summary
- remove the duplicate too-lenient threshold constant and reuse the canonical CRAP threshold
- keep threshold warning behavior unchanged while making the shared boundary explicit
- add focused tests for the default-boundary and above-boundary warning cases

## Verification
- npm run build
- npm test
- npm run crap-typescript-check

Closes #114

### CODE REVIEW

File: packages/core/src/constants.ts [hunk 1]
```diff
@@ -15,8 +14,8 @@ export function thresholdWarning(value: number): string {
   if (value < 4.0) {
     return `Warning: CRAP threshold below 4.0 is likely too noisy. ${thresholdRecommendation()}`;
   }
-  if (value > TOO_LENIENT_THRESHOLD) {
-    return `Warning: CRAP threshold above ${TOO_LENIENT_THRESHOLD.toFixed(1)} is too lenient even for hard gates. ${thresholdRecommendation()}`;
+  if (value > CRAP_THRESHOLD) {
+    return `Warning: CRAP threshold above ${CRAP_THRESHOLD.toFixed(1)} is too lenient even for hard gates. ${thresholdRecommendation()}`;
```
[the remark this answers is not in the document]
Comment (fabian-barney) [reply packages/core/src/constants.ts:19]: Valid. Fixed on `ce42491` by deriving the recommendation text from `CRAP_THRESHOLD` and the internal implementation-target constant instead of hardcoding `8.0` in the guidance string. The warning text stays behaviorally identical today, but it no longer drifts if the shared thresholds change later. I reran `npm test` and `npm run crap-typescript-check` locally after the change.
